internal/config/config.go

```diff
@@ -111,8 +116,18 @@ func (conf *Config) SelectOrganization(org string, inGitRepo bool) error {
 func (conf *Config) APIToken() string {
 	slug := conf.OrganizationSlug()
 	key := fmt.Sprintf("organizations.%s.api_token", slug)
+
+	// Check for deprecated BK_ACCESS_TOKEN and warn if used
+	legacyToken := os.Getenv("BK_ACCESS_TOKEN")
```

Comment:
Could it lead to a race condition when `sync.One` is a global, but warning logic depends on env variables that can change between calls?